qiita_db/test/test_processing_job.py

```diff
@@ -492,7 +492,7 @@ def test_complete_success(self):
         job.complete(True, artifacts_data=artifacts_data)
         self._wait_for_job(job)
         # Retrieve the job that is performing the validation:
-        val_job = qdb.processing_job.ProcessingJob(job.step.rsplit(" ", 1)[-1])
+        val_job = list(job.get_validator_jobs)[-1]
```

Comment:
Why is this checking only the last job?

Reply:
Not sure, that's the test it was before ...

Reply:
haha ok.